Bug: 森空岛同一个账户绑定第二个方舟账号上去后，个人信息无法正常获取

### 确认事项

- [x] 我已仔细阅读并了解上述注意事项。
- [x] 我已使用最新版本测试过，确认问题依旧存在。
- [x] 我确定在 [GitHub Issues](../issues) 中没有相同或相似的问题。

### NoneBot 版本

onebot v11

### NoneBot skland 插件版本或 Commit ID

skland 0.5.5

### 协议端

llonebot 6.1.1

### 描述问题

在此前，同一个qq账号与绑定的森空岛账号发出sk指令能正常获取方舟信息
在绑定了个b服方舟账号上去后，bot发送信息
"[at]user接口请求失败,获取绑定角色失败: [Errno 11001] getaddrinfo failed",
"[at]user接口请求失败,获取绑定角色失败: "
不发送信息的情况则是下面提供的日志


### 复现步骤

尝试使用skland char update，一样
其他群友可以正常获取信息
我绑上去的第二个账号可正常参与签到，但是个人信息获取直接暴毙了，肉鸽是获取森空岛设定的默认账号，可正常运作

### 期望的结果

可在sk指令，指定获取一个账号的信息

### 截图或日志

```11-07 13:08:39 [ERROR] nonebot | Running AlconnaMatcher(type='', command=skland::skland, module=zhenxun.plugins.nonebot_plugin_skland, lineno=114) failed.
Traceback (most recent call last):
  File "C:\RensoCO2\bot.py", line 19, in <module>
    nonebot.run()
  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-Itvr2_kN-py3.10\lib\site-packages\nonebot\__init__.py", line 337, in run
    get_driver().run(*args, **kwargs)
  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-Itvr2_kN-py3.10\lib\site-packages\nonebot\drivers\fastapi.py", line 187, in run
    uvicorn.run(
  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-Itvr2_kN-py3.10\lib\site-packages\uvicorn\main.py", line 580, in run
    server.run()
  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-Itvr2_kN-py3.10\lib\site-packages\uvicorn\server.py", line 67, in run
    return asyncio.run(self.serve(sockets=sockets))
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\base_events.py", line 628, in run_until_complete
    self.run_forever()
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\windows_events.py", line 316, in run_forever
    super().run_forever()
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\base_events.py", line 595, in run_forever
    self._run_once()
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\base_events.py", line 1881, in _run_once
    handle._run()
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-Itvr2_kN-py3.10\lib\site-packages\nonebot\utils.py", line 259, in run_coro_with_shield
    return await coro
  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-Itvr2_kN-py3.10\lib\site-packages\nonebot\message.py", line 508, in check_and_run_matcher
    await _run_matcher(
> File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-Itvr2_kN-py3.10\lib\site-packages\nonebot\message.py", line 460, in _run_matcher
    await matcher.run(bot, event, state, stack, dependency_cache)
  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-Itvr2_kN-py3.10\lib\site-packages\nonebot\internal\matcher\matcher.py", line 926, in run
    await self.simple_run(bot, event, state, stack, dependency_cache)
  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-Itvr2_kN-py3.10\lib\site-packages\nonebot\internal\matcher\matcher.py", line 863, in simple_run
    await handler(
  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-Itvr2_kN-py3.10\lib\site-packages\nonebot\dependencies\__init__.py", line 113, in __call__
    return await cast(Callable[..., Awaitable[R]], self.call)(**values)
  File "C:\RensoCO2\zhenxun\plugins\nonebot_plugin_skland\__init__.py", line 227, in _
    info = await get_character_info(user, str(ark_characters.uid))
  File "C:\RensoCO2\zhenxun\plugins\nonebot_plugin_skland\utils.py", line 89, in wrapper
    return await func(user, *args, **kwargs)
  File "C:\RensoCO2\zhenxun\plugins\nonebot_plugin_skland\utils.py", line 59, in wrapper
    return await func(user, *args, **kwargs)
  File "C:\RensoCO2\zhenxun\plugins\nonebot_plugin_skland\__init__.py", line 216, in get_character_info
    return await SklandAPI.ark_card(CRED(cred=user.cred, token=user.cred_token), uid)
  File "C:\RensoCO2\zhenxun\plugins\nonebot_plugin_skland\api\request.py", line 137, in ark_card
    return ArkCard(**response.json()["data"])
  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-Itvr2_kN-py3.10\lib\site-packages\pydantic\main.py", line 253, in __init__
    validated_self = self.__pydantic_validator__.validate_python(data, self_instance=self)
pydantic_core._pydantic_core.ValidationError: 3 validation errors for ArkCard
building.meeting
  Input should be a valid dictionary or instance of Meeting [type=model_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.11/v/model_type
building.hire
  Input should be a valid dictionary or instance of Hire [type=model_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.11/v/model_type
building.training
  Input should be a valid dictionary or instance of Training [type=model_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.11/v/model_type
```
这是bot不发送获取信息失败情况下的日志

### 参与贡献

- [ ] 我有足够的时间和能力，愿意为此提交 PR 来修复问题。

```11-08 16:31:55 [ERROR] nonebot | Running AlconnaMatcher(type='', command=skland::skland, module=zhenxun.plugins.nonebot_plugin_skland, lineno=114) failed.
Traceback (most recent call last):
  File "C:\RensoCO2\bot.py", line 19, in <module>
    nonebot.run()
  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-Itvr2_kN-py3.10\lib\site-packages\nonebot\__init__.py", line 337, in run
    get_driver().run(*args, **kwargs)
  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-Itvr2_kN-py3.10\lib\site-packages\nonebot\drivers\fastapi.py", line 187, in run
    uvicorn.run(
  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-Itvr2_kN-py3.10\lib\site-packages\uvicorn\main.py", line 580, in run
    server.run()
  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-Itvr2_kN-py3.10\lib\site-packages\uvicorn\server.py", line 67, in run
    return asyncio.run(self.serve(sockets=sockets))
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\base_events.py", line 628, in run_until_complete
    self.run_forever()
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\windows_events.py", line 316, in run_forever
    super().run_forever()
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\base_events.py", line 595, in run_forever
    self._run_once()
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\base_events.py", line 1881, in _run_once
    handle._run()
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-Itvr2_kN-py3.10\lib\site-packages\nonebot\utils.py", line 259, in run_coro_with_shield
    return await coro
  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-Itvr2_kN-py3.10\lib\site-packages\nonebot\message.py", line 508, in check_and_run_matcher
    await _run_matcher(
> File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-Itvr2_kN-py3.10\lib\site-packages\nonebot\message.py", line 460, in _run_matcher
    await matcher.run(bot, event, state, stack, dependency_cache)
  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-Itvr2_kN-py3.10\lib\site-packages\nonebot\internal\matcher\matcher.py", line 926, in run
    await self.simple_run(bot, event, state, stack, dependency_cache)
  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-Itvr2_kN-py3.10\lib\site-packages\nonebot\internal\matcher\matcher.py", line 863, in simple_run
    await handler(
  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-Itvr2_kN-py3.10\lib\site-packages\nonebot\dependencies\__init__.py", line 113, in __call__
    return await cast(Callable[..., Awaitable[R]], self.call)(**values)
  File "C:\RensoCO2\zhenxun\plugins\nonebot_plugin_skland\__init__.py", line 227, in _
    info = await get_character_info(user, str(ark_characters.uid))
  File "C:\RensoCO2\zhenxun\plugins\nonebot_plugin_skland\utils.py", line 83, in wrapper
    return await func(user, *args, **kwargs)
  File "C:\RensoCO2\zhenxun\plugins\nonebot_plugin_skland\utils.py", line 59, in wrapper
    return await func(user, *args, **kwargs)
  File "C:\RensoCO2\zhenxun\plugins\nonebot_plugin_skland\__init__.py", line 216, in get_character_info
    return await SklandAPI.ark_card(CRED(cred=user.cred, token=user.cred_token), uid)
  File "C:\RensoCO2\zhenxun\plugins\nonebot_plugin_skland\api\request.py", line 139, in ark_card
    return ArkCard(**response.json()["data"])
  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-Itvr2_kN-py3.10\lib\site-packages\pydantic\main.py", line 253, in __init__
    validated_self = self.__pydantic_validator__.validate_python(data, self_instance=self)
pydantic_core._pydantic_core.ValidationError: 3 validation errors for ArkCard
building.meeting
  Input should be a valid dictionary or instance of Meeting [type=model_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.11/v/model_type
building.hire
  Input should be a valid dictionary or instance of Hire [type=model_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.11/v/model_type
building.training
  Input should be a valid dictionary or instance of Training [type=model_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.11/v/model_type
```
绑定的时候好像是没开基建（（
但是现在开了基建，也执行了sk char update
返回上面的日志
![Image](https://github.com/user-attachments/assets/f2eb6e59-7962-47f7-9145-da713a08a844)

--- a/nonebot_plugin_skland/schemas/ark_models/building.py
+++ b/nonebot_plugin_skland/schemas/ark_models/building.py
@@ -41,9 +41,9 @@ class Building(BaseModel):
     manufactures: list[Manufacture]
     tradings: list[Trading]
     dormitories: list[Dormitory]
-    meeting: Meeting
-    hire: Hire
-    training: Training
+    meeting: Meeting | None = None
+    hire: Hire | None = None
+    training: Training | None = None
     labor: Labor
     furniture: Furniture
     control: Control

